_includes/components/search_header.html

```diff
@@ -1,23 +1,22 @@
-<div class="search">
-  <form class="search-input-wrap" role="search" action="{{ site.baseurl }}/search/" method="get">
+<search class="search">
```

Comment:
Do you think this has sufficient compatibility? The `search` element has only been supported on Safari since version 17, so older Macs & iOS devices may not get the landmark role from the element. Other browsers are on similar timelines, but Safari is unique in that devices that can't be updated to more recent OS also can't update their browsers.

Reply:
I did this because Axe complained that the search functionality was not in a landmark and I thought [`<search>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/search) was a good option. 
I will change it into `<section aria-label="Search">` and add the role search back on the form element